### ISSUE TITLE
Enhancement: The new chart editor sucks change it back

### ISSUE THREAD

**Issue by NerdiestAlpaca** [KadeArchive/Kade-Engine#1470]
New chart editor takes too long to load, everything being there at once is sensory overload/confusing, can’t even swap sections properly, what were you trying to do with this it’s awful

**Comment by KlutchDj** [comment]
this isnt really doing anything

**Comment by DamianBossPL** [comment]
I agree that they should change it. Note placement is crappy when you change bpm. On 100 (Bopeebo) it is fine but something other than 100 messes everything up. They should change it back!

**Comment by Rainbeon** [comment]
The new chart editor has some good ideas but not being able to move sections, and not being able to have charts in 1/16 like usual bog it down (closest to 1/16 is charting in 1/15 and 1/18, which is really weird). I'd be happy if we just got the old section navigation system and 1/16 steps back
